### DISCUSSION TITLE
Adding a "timesUsed" value to the 2sxc API

### DISCUSSION THREAD

**Discussion by Chris-Behling** [2sic/2sxc#3374] [Ideas]
We often create repeatable content that can be reused on numerous pages throughout a website. A call-to-action that links to a single place is a great example where you may have one item showing up on dozens of pages.

One problem this can cause for content editors is that they may not realize what they are editing is being used in more than one page/place. The admin panel for 2sxc can help you identify **if** a piece of content is used elsewhere, but it doesn't do a great job at showing you where (unlike views, which is much easier).

Would it be possible to add a new parameter to the 2sxc API, which displays a count for the number of times a particular content type is being used?  Our plan for this is to use a formula to display an infobox if the content being edited is used in more than one place. This can help inform the user, but still allow them to continue making their changes. 

Alternatively, a built-in mechanism that can display this would solve all of our problems.

```
{
    "URL": "https://www.google.com",
    "Target": "_blank",
    "DisplayText": "Visit Google",
    "InternalTitle": "Link to Google",
    "Id": 220,
    "Guid": "15e733bb-5b8c-4512-973f-09c975be2db7",
    "Title": "Link to Google",
    "Modified": "2024-05-06T16:48:48Z",
    "Created": "2021-11-05T13:41:51Z",
    "_2sxcEditInformation": {
        "entityId": 220,
        "title": "Link to Google ",
        "isPublished": true
    }
}
```

**Comment by iJungleboy** [comment]
This does sound like a good idea, with some difficulties.

Eg. how something is used is not always "object is child of other object" since other mechanisms can also show it. 

But I do understand that some indication of reuse would be good. 

ATM there is no built in mechanism and I'm also not sure how to do it for the sake of a standardized built-in solution. 

Are you concerned about the 

1. current item being edited
2. other items being selected in pickers?

**Reply by Chris-Behling** [reply]
More concerned about when the current item is being edited. 
Less worried about the selection of the item through a picker.
